.github/actions/setup-helm-docs/action.yml

```diff
@@ -4,11 +4,13 @@ inputs:
   version:
     description: 'Version of helm-docs'
     required: false
-    default: 1.13.1
 runs:
   using: "composite"
   steps:
+    - shell: bash
+      run: |
+        echo "helm-docs-version=_${{ inputs.version }}_"
```

Comment:
Do we wanna keep this echo?

Reply:
no, good catch, test leftovers, removing it now